src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -404,6 +405,19 @@ export default class CanvasController {
     }
 
     /**
+     * Fill entire scope with the background color specified in the current theme
+     *
+     * @for CanvasController
+     * @method canvas_fill_background
+     * @param cc {Object} HTML5 canvas context
+     */
+    canvas_fill_background(cc) {
+        cc.fillStyle = this.theme.SCOPE.BACKGROUND;
+
+        cc.fillRect(0, 0, this.canvas.size.width, this.canvas.size.height);
```

Comment:
Yeah, that terrifies me a little. And we have that canvas refactor issue already, so I figure it can be dealt with then. But I'll drop a todo at the very least.